src/webpack/chunkSplitting.js

```diff
@@ -1,53 +1,57 @@
-const pkgUp = require("pkg-up").sync();
+export const hasExternalizedModuleViaJson = (
+  moduleResource,
+  manifestName,
+  interleave
+) => {
+  if (!moduleResource || !interleave) return;
 
-let packageJson;
-if (pkgUp) {
-  // eslint-disable-next-line import/no-dynamic-require
-  packageJson = require(pkgUp);
-}
-
-export const hasExternalizedModuleViaJson = moduleResource => {
-  const interleaveMap = packageJson.interleave;
-  if (!moduleResource || !interleaveMap) return;
-  const interleaveKeys = Object.keys(packageJson.interleave || {});
+  const interleaveKeys = Object.keys(interleave);
 
   if (interleaveKeys) {
     const foundMatch = interleaveKeys.find(item =>
       moduleResource.includes(item)
     );
-    return interleaveMap[foundMatch] || false;
+    return interleave[foundMatch] || false;
   }
 };
 
-export const hasExternalizedModuleViaJson2 = moduleResource => {
```

Comment:
one thing, I forgot to remove one of these functions. if `hasExternalizedModuleViaJson2` looks the same as `hasExternalizedModuleViaJson` then you can delete one of them and have everything just use the one function. I believe I intended on deleting one and just using a single function `hasExternalizedModuleViaJson2` is newer so it could replace `hasExternalizedModuleViaJson`